app/helpers/users_helper.rb

```diff
@@ -1,2 +1,3 @@
 module UsersHelper
+  
 end
```

Comment:
This right here is the greatest fallback of using scaffolds. I find myself spending more time cleaning up after it than I would if I just created the files/methods i needed by hand.

Anyways, @dkniffin is right, please clean-up all scaffold-generated unused code